lib/generators/boring/sentry/install/install_generator.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Boring
+  module Sentry
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Sentry to the app'
+
+      class_option :use_env_variable, type: :boolean, aliases: '-ev',
+                   desc: 'Use ENV variable for devise_jwt_secret_key. By default Rails credentials will be used.' 
+
+      class_option :breadcrumbs_logger, type: :array, aliases: '-bl',
+                   desc: 'Set the breadcrumbs logger. By default [:active_support_logger, :http_logger] will be used.'
+
```

Comment:
Let's add "default: " with default values as well

you can see https://github.com/TruemarkDev/boring_generators/blob/e034ce5e124a076c2f036a8f881696b8279aa1e4/lib/generators/boring/rswag/install/install_generator.rb on how we are using default values with class_option